### PR TITLE
Allow settings API_HOST and STATIC_HOST instead of HOST in dfx.json

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

When we fetch data from the aggregator canister, this does not go through a candid API.
This allows us to certify the data without using an update call.
With the domain migration, candid API calls will have to go through a different domain.
So we need to be able to specific separate domains for candid API calls vs. static content fetches.

# Changes

Change config.sh to read API_HOST or STATIC_HOST depending on whether the host is used for candid API calls or static content fetches. Both fall back on HOST if not set.

# Tests

Set different values for API_HOST and STATIC_HOST on medium07 and noticed the difference in the `.env` file.
The `.env` file is the same as before when the `dfx.json` file is unchanged but the new hosts are used when present.
Also verified in the browser console that the failed aggregator canister call goes to the new URL.

### CODE REVIEW

**Review by lmuntaner — CHANGES_REQUESTED** [review]
Minor change, thanks!

I realized you only expose `HOST` as `VITE_HOST`. For now this is enough. Yet, we might use both `API_HOST` and `STATIC_HOST` in the frontend.

For example, we have a URL `GOVERNANCE_CANISTER_RAW_URL` created in the frontend. Althuogh instead of using the `STATIC_HOST` in the frontend, I'd rather move `GOVERNANCE_CANISTER_RAW_URL` to the environment. What do you think?

File: config.sh [hunk 1]
```diff
@@ -44,7 +74,7 @@ local_deployment_data="$(
   : "Try to find the internet_identity URL"
   : "- may be deployed locally"
   IDENTITY_SERVICE_URL="$(
-    canister_url_from_id "$(dfx canister --network "$DFX_NETWORK" id internet_identity 2>/dev/null || true)"
+    canister_api_url_from_id "$(dfx canister --network "$DFX_NETWORK" id internet_identity 2>/dev/null || true)"
```
[the remark this answers is not in the document]
Comment (dskloetd) [reply config.sh]: Done

**Comment by dskloetd** [comment]
I agree it makes sense to move GOVERNANCE_CANISTER_RAW_URL to config.sh.
You're not asking me to do that in this PR, right?

**Comment by lmuntaner** [comment]
> I agree it makes sense to move GOVERNANCE_CANISTER_RAW_URL to config.sh. You're not asking me to do that in this PR, right?

Yes, correct. Not in this PR.

**Review by lmuntaner — APPROVED** [review]

**Pull request merged**